docs/devguides/dependency_mgmt.md

```diff
@@ -189,15 +189,15 @@ This is how we achieve that:
 
    ```js
    // Load JS or CSS script if not loaded already
-   Components.loadScript("js", '<script src="/abc/xyz/script.js">');
-   Components.loadScript("css", '<link href="/abc/xyz/style.css">');
+   Components.loadJs('<script src="/abc/xyz/script.js">');
```

Comment:
The JS function `loadScript` was pratically two functions and a switch, so I've split it into two explicit functions, `loadJs` and `loadCss`